src/main/cpp/t09_spiral.cpp

```diff
@@ -17,11 +17,59 @@
 
 #include "t09_spiral.h"
 #include <iostream>
-
-
 using namespace std;
-
-int t09_spiral() {
-    
+int t09_spiral() { 
+   int n, m, c = 1, i = 0, j = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'j' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4488444)